lib/views/general/_responsive_topnav.html.erb

```diff
@@ -14,7 +14,7 @@
         </li>
 
 
-        <li><%= link_to _("Chi Siamo"), ' http://www.dirittodisapere.it/chi-siamo/' %></li>
+        <li><%= link_to "Chi Siamo", ' http://www.dirittodisapere.it/chi-siamo/' %></li>
         <li class="<%= 'selected' if params[:controller] == 'help' %>">
           <%= link_to _("Help"), help_about_path %>
         </li>
```

Comment:
Ugh - CSS regression. Have already had to fix this for frontpage. Good catch, thanks!